engine-precompiles/src/native.rs

```diff
@@ -29,6 +34,9 @@ mod costs {
     // TODO(#51): Determine the correct amount of gas
     pub(super) const FT_TRANSFER_GAS: Gas = 100_000_000_000_000;
 
+    #[cfg(feature = "error_refund")]
+    pub(super) const REFUND_ON_ERROR_GAS: Gas = 60_000_000_000_000;
```

Comment:
Is this an estimation?

Reply:
Can you add a TODO message to accurately measure the amount of gas? Then add in an issue so that we can track it.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/311/commits/5678aba5f81b8a8940af14bed1a8dc3382354bf0